PSKoans/Koans/Introduction/AboutCmdletVerbs.Koans.ps1

```diff
@@ -109,29 +132,31 @@ Describe "Basic Verbs" {
 
     Context "Set" {
         <#
-            Cmdlets with the Set verb will overwrite information that already exists.
+            Cmdlets with the Set verb will overwrite information that already
+            exists.
 
-            If you followed the example in the 'New' and 'Add' tests, you can use set to do
-            something to your text file:
+            If you followed the example in the 'New' and 'Add' tests, you can
+            use set to do something to your text file:
 
                 $Path = "YOUR PATH.txt"
-                Set-Content -Path $Path -Value "Sir Kill-A-Lot is the best house robot because of reasons."
-
-            Before continuing, run this command several times. See what happens, is it the
-            result you expected?
+                $text = "Sir Kill-A-Lot is the best house robot because of reasons."
+                Set-Content -Path $Path -Value $text
 
-            You'll see that there's only one line of text in the file. This is because the Set-Content command
-            will overwrite information that's already there.
+            Before continuing, run this command several times. See what happens;
+            is it the result you expected?
 
-            Some Set-* cmdlets require the instance to already be present for you to change it;
-            you'll need to use a New-* cmdlet first to create an instance before you can
-            overwrite information within it.
+            You'll see that there's only one line of text in the file. This is
+            because the Set-Content command will overwrite information that's
+            already there.
 
-            A common example one may see working with Office 365 is with calendar permissions.
-            If a user already has some permissions configured, you can use you use "Set-MailboxFolderPermission"
-            to change the user's permissions. However, attempting to use an Add cmdlet will fail;
-            Set-* commands overwrite data, but often can only do so if it already exists.
+            Some Set-* cmdlets require the instance to already be present for
+            you to change it; you'll need to use a New-* cmdlet first to create
+            an instance before you can overwrite information within it.
```

Comment:
An example of a cmdlet requiring the combination of New and Set may be helpful at this point, because we earlier established that Add will create something if does not exist, which does not apply for Set. Just to remind everyone.

Reply:
Do you have any examples? 🤔 

Most common cases follow the New-Item / Set-Content behaviour, which is that the target is created if it doesn't exist yet. 😕

Reply:
The example just below this is somewhat accurate, but then again not everyone has or will need to work with office 365 cmdlets 😅

Reply:
Okay, did not get the notification in time, sorry about that. Looked at a few potential candidates, but you are right, most follow New-Item / Set-Content. If I encunter one again I'll update, for now I'm blank.